manifests/o/Obsidian/Obsidian/1.5.3/Obsidian.Obsidian.installer.yaml

```
@@ -0,0 +1,31 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Obsidian.Obsidian
+PackageVersion: 1.5.3
+InstallerType: nullsoft
+InstallModes:
+- silent
+UpgradeBehavior: install
+FileExtensions:
+- md
+ReleaseDate: 2023-12-26
+Installers:
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://github.com/obsidianmd/obsidian-releases/releases/download/v1.5.3/Obsidian.1.5.3.exe
+  InstallerSha256: BFBB6BE48E545AF81D6EA0C66FF8F41D19FCD1AFC21499D44FA89C9F1241BF42
+- Architecture: x86
+  Scope: user
+  InstallerUrl: https://github.com/obsidianmd/obsidian-releases/releases/download/v1.5.3/Obsidian.1.5.3-32.exe
+  InstallerSha256: BFBB6BE48E545AF81D6EA0C66FF8F41D19FCD1AFC21499D44FA89C9F1241BF42
+- Architecture: x64
+  Scope: machine
+  InstallerUrl: https://github.com/obsidianmd/obsidian-releases/releases/download/v1.5.3/Obsidian.1.5.3-allusers.exe
+  InstallerSha256: 6BAF021A476B3731242DDC9B3D5B4A6006BC684D70FCDDA1EB9FF6889DA82319
+- Architecture: arm64
+  Scope: user
+  InstallerUrl: https://github.com/obsidianmd/obsidian-releases/releases/download/v1.5.3/Obsidian.1.5.3-arm64.exe
+  InstallerSha256: 788AAB21ACFEFF7DEB8F3642BD82AB3D3DEA319183756BD488D4C762E0DF941F

```

I'm seeing these hashes as the actual hashes.

```suggestion
- Architecture: x64
  Scope: user
  InstallerUrl: https://github.com/obsidianmd/obsidian-releases/releases/download/v1.5.3/Obsidian.1.5.3.exe
  InstallerSha256: 6BAF021A476B3731242DDC9B3D5B4A6006BC684D70FCDDA1EB9FF6889DA82319
- Architecture: x86
  Scope: user
  InstallerUrl: https://github.com/obsidianmd/obsidian-releases/releases/download/v1.5.3/Obsidian.1.5.3-32.exe
  InstallerSha256: BFBB6BE48E545AF81D6EA0C66FF8F41D19FCD1AFC21499D44FA89C9F1241BF42
- Architecture: x64
  Scope: machine
  InstallerUrl: https://github.com/obsidianmd/obsidian-releases/releases/download/v1.5.3/Obsidian.1.5.3-allusers.exe
  InstallerSha256: BDD44AEF693BE67200EBEFF815CC2CC8DCDF17E05ED11787EF730FB768F76818
- Architecture: arm64
  Scope: user
  InstallerUrl: https://github.com/obsidianmd/obsidian-releases/releases/download/v1.5.3/Obsidian.1.5.3-arm64.exe
  InstallerSha256: 788AAB21ACFEFF7DEB8F3642BD82AB3D3DEA319183756BD488D4C762E0DF941F
```